R/getQuote.R

```diff
@@ -326,6 +332,8 @@ getQuote.av <- function(Symbols, api.key, ...) {
          "Registration at https://api.tiingo.com/.", call. = FALSE)
   }
 
+  Symbols <- unlist(strsplit(Symbols,';'))
```

Comment:
It's good practice to only have one functional change per commit, so this should be in another commit.